deepdrivemd/models/keras_cvae_stream/config.py

```diff
@@ -0,0 +1,57 @@
+from typing import List, Tuple
+from deepdrivemd.config import MachineLearningTaskConfig
+from pathlib import Path
+
+class KerasCVAEModelConfig(MachineLearningTaskConfig):
```

Comment:
Can subclass existing KerasCVAEModelConfig and add necessary streaming parameters

Reply:
I got rid of all the irrelevant parameters but prefer to keep separate config.py files.

Reply:
Makes sense. Thank you!